docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process where all required resources and dependencies are declared and prefetched within a build system. This strict control ensures that the build system has complete visibility and control over the dependencies needed for the build, including the ability to accurately collect, inquire, analyze, and manage all dependencies during the build process. By restricting external dependencies and implementing a centralized management approach, hermetic builds provide a predictable and reliable build environment.
+
+IMPORTANT: If you do not vendor all the required dependencies in a git repository while isolated from the network, your build may fail. 
```

Comment:
I like how the NOTE below is worded in its context more so than this note. It introduces a good concept -- that additional user work may be needed to enable hermetic builds because of the network isolation. Maybe we can just change what we are trying to say here to stay general.